lib/js/_create/_build-new-package.js

```diff
@@ -143,13 +151,20 @@ module.exports = (config, basePath, globalLicense, answers) => {
 	return [
 		{
 			title: `Create folder ${answers.pkgname}`,
-			task: () => fs.mkdir(`${filePath}/${answers.pkgname}`, error => {
-				if (error) {
-					console.error(error);
+			task: () => {
+				// Create the package folder
+				// Make sure it exists before processing next task
+				try {
+					fs.mkdirSync(`${filePath}/${answers.pkgname}`);
+				} catch (error) {
```

Comment:
Force the creation of the top-level folder to finish before starting the next task